src/environments/environment.development.ts

```diff
@@ -3,8 +3,8 @@ export const environment = {
   apiBasePath: `https://localhost:5001/api`,
   resourceServerId: 'api',
   keycloak: {
-    issuer: 'https://localhost:8443/realms/Klockan',
-    redirectUri: 'https://localhost/',
+    issuer: 'http://localhost:8080/realms/Klockan',
+    redirectUri: 'https://localhost:4000/',
```

Comment:
Same here

Reply:
Fixed in [Restore development environment keycloak variables](https://github.com/JU-DEV-LatamBootcamp/Klockan-Frontend/pull/49/commits/99ed40139c61155193fd99bb019a9c5a0d23f08f) commit.